src/component/addedittimeline/index.tsx

```diff
@@ -186,11 +186,12 @@ function AddEditTimeline(props: AddEditTimelineProps) {
         handleDelete: () => navigate(`/${ventureId}/${timelineId}/delete`),
         handleSave: () => handleSubmit(handleCreate)(),
       }}
-      visibility={{
-        "aria-label": "Timeline Visibility",
-        defaultValue: "private",
-        setVisibility,
-      }}
+      // TODO: Confirm later with the team
+      // visibility={{
+      //   "aria-label": "Timeline Visibility",
+      //   defaultValue: "private",
+      //   setVisibility,
+      // }}
```

Comment:
Waiting for the confirmation from @disresz as I couldn't find this prop anymore in Plasmic

Reply:
You can remove this. We wanted to make this like Google Docs so you only set visibility by clicking on the "Share" button instead of setting it during timeline creation.